test/compilable/triple.d

```diff
@@ -0,0 +1,12 @@
+/* The following arguments to the autotester do not work because it is
+ * determined to add -fPICx, even though the documentation says xERMUTE_ARGSx
+ * with no arguments will prevent it.
+ */
+/* EQUIRED_ARGSx -m64 -target=x64-windows-msvc -mscrtlib=whatever.lib
+ * ERMUTE_ARGSx
```

Comment:
Typo?

Reply:
Nope. The test runner has a buggy parser in it, and will recognize `PERMUTE_ARGS` if it is embedded in another identifier.

Reply:
That should be addressed by #13478.